### PR TITLE
refs#paynow_enroll_report Added loggers files & changed file name

### PR DESCRIPTION
**Pull request by jayreddy519**
### Master Redmine ticket
* (Required!)

### Child Redmine ticket(s)
* ()
* ()

### Local build result

```
bundle exec rake parallel:spec[4]
bundle exec cucumber
```

### Latest rebase/merge tag
*

---

### Data ticket(s) Followup
* (redmine links here - optional)

### Related Pull Requests
* (github links here - optional)

---

#### Peer Review
* (For code review)

#### Functional Testing
* (For testing locally)

#### Deployment
* (for release manager and/or build manager)

### CODE REVIEW

File: script/kaiser_enrollment_list_for_pay_now.rb [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+field_names = %w[First_Name
+                 Last_Name
+                 HBX_ID
+                 Application_Year
+                 Enrollment_hbx_id
+                 Aasm_state]
+
+logger_field_names = %w[Enrollment_ID Backtrace]
+
+file_name = "#{Rails.root}/enrollment_list_for_pay_now.csv"
```
Comment (ymhari) [inline script/kaiser_enrollment_list_for_pay_now.rb:15]: Do you need this ?

File: script/kaiser_enrollment_list_for_pay_now.rb [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+report_file_name = "#{Rails.root}/kaiser_enrollment_list_for_pay_now_#{TimeKeeper.date_of_record.strftime('%m_%d_%Y')}.csv"
+logger_file_name = "#{Rails.root}/kaiser_enrollment_list_for_pay_now_logger_#{TimeKeeper.date_of_record.strftime('%m_%d_%Y')}.csv"
+
+CSV.open(logger_file_name, 'w', force_quotes: true) do |logger_csv|
+  logger_csv << logger_field_names
+  CSV.open(report_file_name, 'w', force_quotes: true) do |report_csv|
+    report_csv << field_names
+    effective_on = TimeKeeper.date_of_record
```
Comment (ymhari) [inline script/kaiser_enrollment_list_for_pay_now.rb:23]: I don't think this is being used.

**Review by ymhari — CHANGES_REQUESTED** [review]

File: script/kaiser_enrollment_list_for_pay_now.rb (hunk 2 above)
Comment (jayreddy519) [reply script/kaiser_enrollment_list_for_pay_now.rb]: Removed this on latest commit.

File: script/kaiser_enrollment_list_for_pay_now.rb (hunk 1 above)
Comment (jayreddy519) [reply script/kaiser_enrollment_list_for_pay_now.rb]: Removed this on latest commit.

**Review by ymhari — APPROVED** [review]

**Pull request merged**